### ISSUE TITLE
Readme has incorrect url for buildpack.

### ISSUE THREAD

**Issue by jimmiebtlr** [RocketChat/heroku-buildpack-meteor#12]
Readme has incorrect url for buildpack.  Seems to still be pointing at AdmitHub

**Comment by sampaiodiego** [comment]
you're right @jimmiebtlr .. thanks for pointing it out. would you like to contribute fixing it? thx
